options/namesys/opts.go

```diff
@@ -72,3 +72,48 @@ func ProcessOpts(opts []ResolveOpt) ResolveOpts {
 	}
 	return rsopts
 }
+
+// defaultRecordEOL specifies the time that the network will cache IPNS
+// records after being published. Records should be re-published before this
+// interval expires. We use the same default expiration as the DHT.
+const defaultRecordEOL = 48 * time.Hour
```

Comment:
It feels useful to expose this default as `DefaultIPNSRecordEOL`, so users of this librayr can detect when we changed it.